docs/api/gds/async.rst

```diff
@@ -0,0 +1,24 @@
+Asynchronous client
+=======================
+
+This is the client to be used for retrieving the GDS information
+in an asynchronous (i.e. non-blocking) fashion.
+
+>>> import asyncio
+>>> from pysdmx.api.gds import AsyncGdsClient
+>>> async def main():
+>>>     gr = AsyncGdsClient()
+>>>     gds_agencies = await gr.get_agencies("BIS")
+>>>     print(gds_agencies)
+>>> asyncio.run(main())
+
+.. important::
+    This client is designed to be used with Python's `asyncio` library.
```

Comment:
Maybe we could add that this is the recommended client to be used when using asynchronous frameworks like `fastapi`?